boards/nxp/mimxrt1060_evk/revision.cmake

```diff
@@ -0,0 +1,11 @@
+if (NOT DEFINED BOARD_REVISION)
+	if (CONFIG_BOARD_MIMXRT1060_EVK)
+		set(BOARD_REVISION "qspi")
+	endif()
+else ()
+	if (NOT (BOARD_REVISION STREQUAL "hyperflash") AND NOT (BOARD_REVISION STREQUAL "qspi"))
+		message(FATAL_ERROR "Invalid board revision")
+	elseif (BOARD_REVISION STREQUAL "hyperflash" AND CONFIG_BOARD_MIMXRT1060_EVKB)
+		message(FATAL_ERROR "hyperflash not supported on RT1060 EVKB")
+	endif()
+endif()
```

Comment:
as above

Reply:
please elaborate, is it just because you dont think it should be using revisions?